components/RadioGroup/RadioGroup.jsx

```diff
@@ -1,27 +1,36 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import Radio from '../Radio';
+import Radio from './Radio.jsx';
+import RadioButton from './RadioButton.jsx';
 
-function getClassName({ color, stacked }) {
+function getClassName({ buttons, color, stacked }) {
   return classNames({
     'radio-teal': color === 'teal',
     'radio-gray': color === 'gray',
+    'radio--buttons': buttons,
     'radio--stacked': stacked,
   });
 }
 
-const RadioGroup = ({ color, items, onChange, selectedIndex, stacked }) => (
+function getRadioComponent({ buttons, onChange, selectedIndex }) {
+  return buttons ?
+    (item, i) => <RadioButton onChange={onChange} index={i} key={i} checked={selectedIndex === i} {...item} /> :
+    (item, i) => <Radio onChange={onChange} index={i} key={i} checked={selectedIndex === i} {...item} />;
```

Comment:
I don't mind defaulting the `key` prop to the item index, but I'm wondering if we can open this up just in case we need to pass in a more unique identifier. https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318